daemon/mgr/image.go

```diff
@@ -150,7 +150,8 @@ func (mgr *ImageManager) ListImages(ctx context.Context, filter ...string) ([]ty
 	for _, img := range imgs {
 		imgCfg, err := img.Config(ctx)
 		if err != nil {
-			return nil, err
+			logrus.Warnf("failed to get image config info during list images: %v", err)
```

Comment:
maybe we should print some WARN info in cli side , like ctr ：
```
#ctr --address /var/run/containerd.sock images ls
WARN[0000] encountered unknown type application/octet-stream; children may not be fetched
WARN[0000] encountered unknown type application/octet-stream; children may not be fetched
```

Reply:
I think it is quite hard to do so in cli side. Since cli is heavily relying on the api, and if the api side does not return the error or warning message, nothing could be done in cli side, I think.